### PR TITLE
UI basic setup

### PR DESCRIPTION
**Pull request by r-sukanya**
basic UI setup

### CODE REVIEW

File: client/scss/main.scss [hunk 1]
```diff
@@ -71,3 +71,31 @@ form button[type="submit"] {
...
+.header {
+    background-color: #2e80a9;
+    height: 20%;
+    text-align: center;
+}
+
+.text-input {
+    margin :20px;
+    .mdl-textfield{
+        width: 100% !important;
```
Comment (sebmarste) [inline client/scss/main.scss]: do we really need important here?

File: client/index.html [hunk 2]
```diff
@@ -7,8 +7,41 @@
...
+<main>
+  <section class="header">
+    <h3>Javascript Technology Seminar Project</h3>
+  </section>
+  <section>
+  <form action="#" class="text-input">
+    <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
+      <input class="mdl-textfield__input" type="text" id="sample3">
+      <label class="mdl-textfield__label" for="sample3">Plot a...</label>
+    </div>
```
Comment (sebmarste) [inline client/index.html]: could we add a submit button as well?

File: client/scss/main.scss (hunk 1 above)
Comment (r-sukanya) [reply client/scss/main.scss]: yes, doesnt work without it.

File: client/index.html (hunk 2 above)
Comment (r-sukanya) [reply client/index.html]: sure

File: client/scss/main.scss (hunk 1 above)
Comment (sebmarste) [reply client/scss/main.scss]: thats weird, for me it does work perfectly without that

**Comment by sebmarste** [comment]
could you please use that code formatting of intellij here as well (so all css classes have the same spacing and so on)

File: client/scss/main.scss (hunk 1 above)
Comment (r-sukanya) [reply client/scss/main.scss]: you were right . I also changed some structure

**Pull request merged**